### PR TITLE
registering our first ciam app

### PR DESCRIPTION
**Pull request by zwitter1**

### CODE REVIEW

File: chrome/fed-modules.json [hunk 1]
```diff
@@ -685,5 +685,17 @@
...
         ]
+    },
+    "iam": {
+      "manifestLocation": "/apps/iam/fed-mods.json",
+      "modules": [
+          {
+              "id": "authentication-policy",
+              "module": "./RootApp",
+              "routes": [
+                  "/iam/authpolicy"
```
Comment (Hyperkid123) [inline chrome/fed-modules.json:696]: @ryelo do we need some special setup in Akamai to allow a new bundle?

**Comment by Hyperkid123** [comment]
@zwitter1 sorry for the delay we had our hackathon week.

**Review by Hyperkid123 — COMMENTED** [review]
@zwitter1 OK I have tested the config (used a different module) and it loads.

We are missing a navigation file for `/iam` bundle. Based on our conversations I understood that there will be a navigation for your applications. Chrome is currently expecting navigation config. We can tell it otherwise.

Can you add a `iam-navigation.json` file to the `/chrome` directory? It can be very simple. Something similar to this:

```JSON
{
  "id": "iam",
  "title": "IAM",
  "navItems": [
      {
          "title": "Authentication policy",
          "appId": "iam",
          "href": "/iam/authpolicy"
      },
  ]
}

```

File: chrome/fed-modules.json (hunk 1 above)
Comment (karelhala) [reply chrome/fed-modules.json:696]: No need to, it's just missing a main.yml and navigation update.

**Comment by zwitter1** [comment]
> @zwitter1 OK I have tested the config (used a different module) and it loads.
> 
> We are missing a navigation file for `/iam` bundle. Based on our conversations I understood that there will be a navigation for your applications. Chrome is currently expecting navigation config. We can tell it otherwise.
> 
> Can you add a `iam-navigation.json` file to the `/chrome` directory? It can be very simple. Something similar to this:
> 
> ```json
> {
>   "id": "iam",
>   "title": "IAM",
>   "navItems": [
>       {
>           "title": "Authentication policy",
>           "appId": "iam",
>           "href": "/iam/authpolicy"
>       },
>   ]
> }
> ```

@Hyperkid123 Added the navigation file to meet the design provided by ciam's ux team

File: chrome/fed-modules.json (hunk 1 above)
Comment (zwitter1) [reply chrome/fed-modules.json:696]: @karelhala Ive tweaked the main.yml but I am uncomfortable with the use of "id's" in console dot and would appreciate if you could review that aspect. I tried to turn the references in my iam-navigation file into references to the "sub-id" from the main.yml

File: chrome/iam-navigation.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+  "id": "iam",
+  "title": "Identity & Access Management",
+  "navItems": [
+    {
+      "title": "User Access",
+      "expandable": true,
+      "routes": [
+        {
+          "appId": "user-access",
```
Comment (Hyperkid123) [inline chrome/iam-navigation.json]: The app id should always be `iam`. It references the module as a whole. The other ids in your fed-modules.json are there to identify the specific entry point.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**